### ISSUE TITLE
"Flicking" gesture is being confused with "Dragging"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#51]
```
"Dragging" and "Flicking" as explained here in Touch Gesture Reference Guide:
http://www.lukew.com/touch/TouchGestureGuide.pdf

What steps will reproduce the problem?
1.Open a Document
2.make a "flick"(swipe)gesture to move the page

What is the expected output? What do you see instead?
When i "flick" in the page i expect that the page get "momentum"(inertia) and 
stay in motion even after i release my finger from screen.
What i see is that after doing the "flick" gesture the page get stuck.
Apparently if you don't do the flick gesture fast and "long" (distance with 
finger in contact with screen) the gesture is confused with a simple "dragging" 
and page simply moves a little as if you were dragging.
This don't happens always, because it depends on how you do the touch gesture.
It is like the "threshold" between flick and drag is changed in ebookdroid

What version of the product are you using? On what operating system?
all versions, from first release to present 1.2.0

What device are you using?
Flytouch 2. 
A 10 inch china tablet with Android 2.1 and a Resistive one-touch screen with 
1024x600 screen resolution

Please provide any additional information below.
You can see this in this video made with ebookdroid first release:
http://www.youtube.com/watch?v=ZSDyAXbz2FY
aprox. in seconds 0:21 ; 0:29 ; 0:32

This is not a big deal but it is a bit annoying sometimes.
Also i have a doubt. I thought that whenever a program put in memory an still 
image(bitmap) or another "whatever still rendered thing", it was up to the 
"Android OS" to scroll and detect touch gestures, etc. But apparently i was 
wrong because the behavior of this program in this regard is different from 
others, even within different version of ebookdroid.

thanks
```

Original issue reported on code.google.com by `lanos...@gmail.com` on 9 Sep 2011 at 6:49

**Comment by GoogleCodeExporter** [comment]
```
>> It is like the "threshold" between flick and drag is changed in ebookdroid

Possibly. It is the most problem part of UI - because too many actors are 
involved:
single touch hanlder, multi-touch handler, curler, scroller, etc.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 9 Sep 2011 at 9:13
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
Update:
In version 1.2.0 this is not happening anymore in djvu documents.
I tested only pdf's and djvu's.
This "problem" only occurs with pdf documents in version 1.2.0.

This is a document example in which this happens(but i tried others and also 
has this issues)
http://www.megaupload.com/?d=YDS5HTOU
```

Original comment by `lanos...@gmail.com` on 16 Sep 2011 at 7:56

**Comment by GoogleCodeExporter** [comment]
```
Please try the 1.2.1-rc2 version.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 3 Oct 2011 at 8:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I tried it but the problem persist.

Maybe is a little better, i don't know, is difficult to say, but definitely the 
"problem" remains.
```

Original comment by `lanos...@gmail.com` on 3 Oct 2011 at 6:01

**Comment by GoogleCodeExporter** [comment]
```
Pls try the 1.2.2-rc2 version
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 4 Oct 2011 at 3:48

**Comment by GoogleCodeExporter** [comment]
```
With 1.2.2-rc2 version is extremely WORST.
Now i can't even get to work the "flick" gesture. No matter how hard i try, 
every gesture i do is detected as a "dragging" gesture. It is impossible for me 
now, to "fling" the page.

PS:What i mean with "fling" is this: A fling gesture is, essentially, leaving 
velocity on an item that was being dragged across a screen. 
source:http://mobile.tutsplus.com/tutorials/android/android-gesture/
Ps2: Also in that page is an example code to get "fling" functional, perhaps 
that can help you. 

Thanks
```

Original comment by `lanos...@gmail.com` on 4 Oct 2011 at 6:26

**Comment by GoogleCodeExporter** [comment]
```
I noticed one more thing.
After i open the document when i do first touch i can get the page to fling.
If then the page flashes to black(see issue 52) from that moment the fling 
gesture is impossible to do.
If the pages don't flashes after the first touch (this is not common usually 
flashes after first touch) then i can fling the page again till the page "black 
flashes"

Conclusion: I can do the fling gesture before the first "black flashes", and 
this flash occur every time, a few moments after i start scrolling so fling is 
not working in practice.

Now issue 51 and 52 are related, really i'm not crazy jaja!!
```

Original comment by `lanos...@gmail.com` on 4 Oct 2011 at 6:56

**Comment by GoogleCodeExporter** [comment]
```
Pls try the 1.2.2-rc3 version.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 5 Oct 2011 at 10:23

**Comment by GoogleCodeExporter** [comment]
```
Pls try the 1.2.2-rc4 version.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 5 Oct 2011 at 3:54

**Comment by GoogleCodeExporter** [comment]
```
Perfect.
This issue was solved in 1.2.2-rc3 version and remain solved in the 1.2.2-rc4 
version.
Thanks
```

Original comment by `lanos...@gmail.com` on 5 Oct 2011 at 9:59
